### PR TITLE
updated static_inventories + WIP group_vars for wingedhelix and betab…

### PR DESCRIPTION
**Pull request by RoanKanninga**
…arrel

### CODE REVIEW

File: group_vars/wingedhelix_cluster/vars.yml [hunk 1]
```diff
@@ -43,14 +43,13 @@ group_subfolder_structure: [
...
-# Disabled production groups, which do not yet exist on Winged Helix during testing phase.
-#
+
+
 #   { group: 'umcg-gap',
 #     lfs: 'tmp07',
 #     mode: "{{ MODE_2770_HARD }}",
 #     owner: 'umcg-gap-ateambot',
-#     subfolders: ['DiagnosticOutput', 'generatedscripts', 'generatedscripts_array', 'logs', 'projects', 'rawdata', 'runs_array', 'Samplesheets/new', 'tmp'],
+#     subfolders: ['DiagnosticOutput', 'generatedscripts','logs', 'projects', 'rawdata', 'Samplesheets', 'tmp'],
```
Comment (pneerincx) [inline group_vars/wingedhelix_cluster/vars.yml:52]:
```suggestion
#     subfolders: ['DiagnosticOutput', 'generatedscripts', 'logs', 'projects', 'rawdata', 'Samplesheets', 'tmp'],
```

File: group_vars/wingedhelix_cluster/vars.yml [hunk 2]
```diff
@@ -43,51 +43,46 @@ group_subfolder_structure: [
...
+     owner: 'umcg-gap-dm',
+     subfolders: ['DiagnosticOutput','logs','samplesheets/new'],
+     machines: "{{ groups['chaperone'] }}" },
+   { group: 'umcg-gap',
+     lfs: 'prm07',
+     mode: "{{ MODE_2750_HARD }}",
+     owner: 'umcg-gap-dm',
+     subfolders: ['concordance/array', 'logs', 'rawdata/array/IDAT','rawdata/array/GTC', 'runs', 'Samplesheets/archive'],
+     machines: "{{ groups['chaperone'] }}" },
+
```
Comment (pneerincx) [inline group_vars/wingedhelix_cluster/vars.yml:66]:
```suggestion
```

File: group_vars/wingedhelix_cluster/vars.yml [hunk 3]
```diff
@@ -43,51 +43,46 @@ group_subfolder_structure: [
...
-#     subfolders: ['concordance', 'logs', 'projects', 'Samplesheets/archive', 'rawdata/ngs', 'trendanalysis'],
-#     machines: "{{ groups['chaperone'] }}" },
-#   { group: 'umcg-genomescan',
-#     lfs: 'tmp07',
-#     mode: "{{ MODE_2770_HARD }}",
-#     owner: 'umcg-genomescan-ateambot',
-#     subfolders: ['logs', 'rawdata', 'runs', 'Samplesheets'],
-#     machines: "{{ groups['user_interface'] }}" },
+
+
```
Comment (pneerincx) [inline group_vars/wingedhelix_cluster/vars.yml:47]:
```suggestion
```

File: single_role_playbooks/pre_deploy_checks.yml [hunk 4]
```diff
@@ -29,7 +29,7 @@
       connection: local
     - name: 'Download dependencies from Ansible Galaxy on the Ansible control host.'
       ansible.builtin.command:
-        cmd: ansible-galaxy install -r requirements.yml --timeout 120
+        cmd: ansible-galaxy install -r requirements.yml #--timeout 120
```
Comment (pneerincx) [inline single_role_playbooks/pre_deploy_checks.yml:32]:
```suggestion
        cmd: ansible-galaxy install -r requirements.yml{% if ansible_version.full is version_compare('2.13', '>=') %} --timeout 120{% endif %}
```

File: group_vars/wingedhelix_cluster/vars.yml [hunk 5]
```diff
@@ -657,22 +686,36 @@ crontabs:
     job: /bin/bash -c "{{ configure_env_in_cronjob }};
          module load NGS_Automated/{{ group_module_versions['umcg-genomescan']['NGS_Automated'] }}-bare;
          notifications.sh -g umcg-genomescan -e"
-  - name: NGS_Automated_pullRawDataFromDS  # Unique ID required to update existing cronjob: do not modify.
+  - name: PullAndProcessGsAnalysisData  # Unique ID required to update existing cronjob: do not modify.
```
Comment (pneerincx) [inline group_vars/wingedhelix_cluster/vars.yml:689]:
```suggestion
  - name: NGS_Automated_PullAndProcessGsAnalysisData  # Unique ID required to update existing cronjob: do not modify.
```

File: group_vars/wingedhelix_cluster/vars.yml [hunk 6]
```diff
@@ -624,21 +639,35 @@ crontabs:
     job: /bin/bash -c "{{ configure_env_in_cronjob }};
          module load NGS_Automated/{{ group_module_versions['umcg-gst']['NGS_Automated'] }}-bare;
          notifications.sh -g umcg-gst -e"
-  - name: NGS_Automated_pullRawDataFromDS  # Unique ID required to update existing cronjob: do not modify.
+  - name: PullAndProcessGsAnalysisData  # Unique ID required to update existing cronjob: do not modify.
```
Comment (pneerincx) [inline group_vars/wingedhelix_cluster/vars.yml:642]:
```suggestion
  - name: NGS_Automated_PullAndProcessGsAnalysisData  # Unique ID required to update existing cronjob: do not modify.
```

File: group_vars/wingedhelix_cluster/vars.yml [hunk 7]
```diff
@@ -24,30 +24,12 @@ extra_easyconfigs_prefix: "{{ easybuild_software_dir }}/easyconfigs/{{ extra_eas
...
-    mode: "{{ MODE_2770_HARD }}",
-    owner: 'umcg-atd-dm',
-    subfolders: ['ConcordanceCheckOutput','ConcordanceCheckSamplesheets/archive', 'logs', 'samplesheets', 'temp', 'trendanalysis'],
-    machines: "{{ groups['chaperone'] }}" },
-  { group: 'umcg-atd',
-    lfs: 'prm07',
-    mode: "{{ MODE_2750_HARD }}",
-    owner: 'umcg-atd-dm',
-    subfolders: ['concordance', 'logs', 'projects', 'Samplesheets/archive', 'rawdata/ngs', 'trendanalysis'],
-    machines: "{{ groups['chaperone'] }}" },
```
Comment (pneerincx) [inline group_vars/wingedhelix_cluster/vars.yml:27]:
```suggestion

```

File: group_vars/wingedhelix_cluster/vars.yml [hunk 8]
```diff
@@ -81,11 +63,37 @@ group_subfolder_structure: [
     subfolders: ['concordance', 'logs', 'projects', 'Samplesheets/archive', 'rawdata/ngs', 'trendanalysis'],
     machines: "{{ groups['chaperone'] }}" },
```
Comment (pneerincx) [inline group_vars/wingedhelix_cluster/vars.yml:65]:
```suggestion

```

File: group_vars/wingedhelix_cluster/vars.yml [hunk 9]
```diff
@@ -81,11 +63,37 @@ group_subfolder_structure: [
...
 
+  { group: 'umcg-genomescan',
+    lfs: 'tmp07',
+    mode: "{{ MODE_2770_HARD }}",
+    owner: 'umcg-genomescan-ateambot',
+    subfolders: ['logs', 'runs', 'Samplesheets','generatedscripts/NGS_DNA', 'projects/NGS_DNA', 'rawdata/ngs', 'Samplesheets/DRAGEN', 'Samplesheets/NGS_DNA'],
+    machines: "{{ groups['user_interface'] }}" },
+
```
Comment (pneerincx) [inline group_vars/wingedhelix_cluster/vars.yml:72]:
```suggestion

```

File: group_vars/wingedhelix_cluster/vars.yml [hunk 10]
```diff
@@ -81,11 +63,37 @@ group_subfolder_structure: [
...
+    owner: 'umcg-atd-dm',
+    subfolders: ['ConcordanceCheckOutput','ConcordanceCheckSamplesheets/archive', 'logs', 'samplesheets', 'temp', 'trendanalysis'],
+    machines: "{{ groups['chaperone'] }}" },
+  { group: 'umcg-atd',
+    lfs: 'prm07',
+    mode: "{{ MODE_2750_HARD }}",
+    owner: 'umcg-atd-dm',
+    subfolders: ['concordance', 'logs', 'projects', 'Samplesheets/archive', 'rawdata/ngs', 'trendanalysis'],
+    machines: "{{ groups['chaperone'] }}" },
+
```
Comment (pneerincx) [inline group_vars/wingedhelix_cluster/vars.yml:91]:
```suggestion

```

File: group_vars/wingedhelix_cluster/vars.yml [hunk 11]
```diff
@@ -100,18 +108,11 @@ group_subfolder_structure: [
     subfolders: ['concordance/array', 'logs', 'rawdata/array/IDAT','rawdata/array/GTC', 'runs', 'Samplesheets/archive'],
     machines: "{{ groups['chaperone'] }}" },
```
Comment (pneerincx) [inline group_vars/wingedhelix_cluster/vars.yml:110]:
```suggestion

```

File: group_vars/wingedhelix_cluster/vars.yml [hunk 12]
```diff
@@ -154,7 +155,7 @@ group_module_versions:
     NGS_Automated: 3.4.0
     ngs-utils: 22.01.1
 crontabs:
-  ######################################################################################################################
+ ######################################################################################################################
```
Comment (pneerincx) [inline group_vars/wingedhelix_cluster/vars.yml:158]:
```suggestion
  ######################################################################################################################
```

Comment (pneerincx) [reply group_vars/wingedhelix_cluster/vars.yml:158]: Please keep comments correctly indented.

File: group_vars/wingedhelix_cluster/vars.yml [hunk 13]
```diff
@@ -232,7 +233,7 @@ crontabs:
     minute: '*/10'
     job: /bin/bash -c "{{ configure_env_in_cronjob }};
          module load NGS_Automated/{{ group_module_versions['umcg-atd']['NGS_Automated'] }}-bare;
-         copyRawDataToTmp.sh -g umcg-atd -p NGS_DNA -s wh-porch+wingedhelix"
+         copyRawDataToTmp.sh -g umcg-atd -s wh-porch+wingedhelix -p DRAGEN -f run01.processGsRawData.finished"
```
[the remark this answers is not in the document]
Comment (pneerincx) [reply group_vars/wingedhelix_cluster/vars.yml:236]: Same logic can be used in other places too and fetches the correct machine names from the static_inventory as opposed to using hard-coded values. Makes it easier to copy-paste the same cronjob to group_vars for similar GD machines.

File: group_vars/copperfist_cluster/vars.yml [hunk 14]
```diff
@@ -24,134 +24,135 @@ extra_easyconfigs_prefix: "{{ easybuild_software_dir }}/easyconfigs/{{ extra_eas
...
+#    owner: 'umcg-genomescan-ateambot',
+#    subfolders: ['logs', 'runs', 'Samplesheets','generatedscripts/NGS_DNA', 'projects/NGS_DNA', 'rawdata/ngs', 'Samplesheets/DRAGEN', 'Samplesheets/NGS_DNA', 'tmp/NGS_DNA'],
+#    machines: "{{ groups['user_interface'] }}" },
   { group: 'umcg-atd',
-    lfs: 'tmp07',
+    lfs: 'tmp06',
     mode: "{{ MODE_2770_HARD }}",
     owner: 'umcg-atd-ateambot',
-    subfolders: ['concordance/array', 'concordance/jobs','concordance/logs','concordance/ngs','concordance/results', 'concordance/samplesheets','concordance/tmp', 'generatedscripts', 'logs', 'projects', 'rawdata/ngs', 'Samplesheets/new', 'tmp', 'trendanalysis'],
+    subfolders: ['concordance/array', 'concordance/jobs', 'concordance/logs', 'concordance/ngs', 'concordance/results', 'concordance/samplesheets', 'concordance/tmp', 'generatedscripts/NGS_Demultiplexing', 'generatedscripts/NGS_DNA', 'logs', 'projects/NGS_DNA','projects/NGS_Demultiplexing', 'rawdata/ngs', 'Samplesheets/NGS_DNA', 'Samplesheets/archive', 'Samplesheets/NGS_Demultiplexing', 'tmp/NGS_DNA', 'tmp/NGS_Demultiplexing' , 'trendanalysis'],
```
Comment (kdelange) [inline group_vars/copperfist_cluster/vars.yml:76]: subfolders trendnalayse: darwin, dragen, database, rawdata, projects, reports, tmp

File: group_vars/copperfist_cluster/vars.yml [hunk 15]
```diff
@@ -310,461 +332,484 @@ crontabs:
...
+  #   job: /bin/bash -c "{{ configure_env_in_cronjob }};
+  #        module load NGS_Automated/{{ group_module_versions['umcg-gd']['NGS_Automated'] }}-bare;
+  #        copyProjectDataToPrm.sh -g umcg-gd -p NGS_DNA"
+  # - name: NGS_Automated_trendAnalysis  # Unique ID required to update existing cronjob: do not modify.
+  #   user: umcg-gd-ateambot
+  #   machines: "{{ groups['user_interface'] | default([]) }}"
+  #   minute: '*/10'
+  #   job: /bin/bash -c "{{ configure_env_in_cronjob }};
+  #        module load NGS_Automated/{{ group_module_versions['umcg-gd']['NGS_Automated'] }}-bare;
+  #        trendanalysis.sh -g umcg-gd"
```
Comment (kdelange) [inline group_vars/copperfist_cluster/vars.yml:435]: trendAnalyse.sh, hoeft dus maar 1x per week

File: group_vars/wingedhelix_cluster/vars.yml [hunk 16]
```diff
@@ -24,11 +24,56 @@ extra_easyconfigs_prefix: "{{ easybuild_software_dir }}/easyconfigs/{{ extra_eas
...
+   mode: "{{ MODE_2770_HARD }}",
+   owner: 'umcg-genomescan-ateambot',
+   subfolders: ['logs', 'runs', 'Samplesheets','generatedscripts/NGS_DNA', 'projects/NGS_DNA', 'rawdata/ngs', 'Samplesheets/DRAGEN', 'Samplesheets/NGS_DNA', 'tmp/NGS_DNA'],
+   machines: "{{ groups['user_interface'] }}" },
   { group: 'umcg-atd',
     lfs: 'tmp07',
     mode: "{{ MODE_2770_HARD }}",
     owner: 'umcg-atd-ateambot',
-    subfolders: ['concordance/array', 'concordance/jobs','concordance/logs','concordance/ngs','concordance/results', 'concordance/samplesheets','concordance/tmp', 'generatedscripts', 'logs', 'projects', 'rawdata/ngs', 'Samplesheets/new', 'tmp', 'trendanalysis'],
+    subfolders: ['concordance/array', 'concordance/jobs', 'concordance/logs', 'concordance/ngs', 'concordance/results', 'concordance/samplesheets', 'concordance/tmp', 'generatedscripts/NGS_Demultiplexing', 'generatedscripts/NGS_DNA', 'logs', 'projects/NGS_DNA','projects/NGS_Demultiplexing', 'rawdata/ngs', 'Samplesheets/NGS_DNA', 'Samplesheets/archive', 'Samplesheets/NGS_Demultiplexing', 'tmp/NGS_DNA', 'tmp/NGS_Demultiplexing' , 'trendanalysis'],
```
Comment (kdelange) [inline group_vars/wingedhelix_cluster/vars.yml:76]: trendanalyse sub folders: darwin, dragen, database, reports, projects, rawdata, reports, tmp

File: group_vars/wingedhelix_cluster/vars.yml [hunk 17]
```diff
@@ -317,389 +339,426 @@ crontabs:
...
          module load NGS_Automated/{{ group_module_versions['umcg-gd']['NGS_Automated'] }}-bare;
-         copyProjectDataToPrm.sh -g umcg-gd"
+         copyProjectDataToPrm.sh -g umcg-gd -p NGS_DNA"
+  - name: NGS_Automated_trendAnalysis  # Unique ID required to update existing cronjob: do not modify.
+    user: umcg-gd-ateambot
+    machines: "{{ groups['user_interface'] | default([]) }}"
+    minute: '*/10'
+    job: /bin/bash -c "{{ configure_env_in_cronjob }};
+         module load NGS_Automated/{{ group_module_versions['umcg-gd']['NGS_Automated'] }}-bare;
+         trendanalysis.sh -g umcg-gd"
```
Comment (kdelange) [inline group_vars/wingedhelix_cluster/vars.yml:435]: trendAnalyse.sh

File: group_vars/copperfist_cluster/vars.yml [hunk 18]
```diff
@@ -310,461 +332,484 @@ crontabs:
...
-    disabled: true
-    job: /bin/bash -c "/bin/mv -v "/groups/umcg-patho/tmp07/Samplesheets/new/"*".csv" "/groups/umcg-patho/tmp07/Samplesheets/""
+  # - name: NGS_Automated_moveAndCheckSamplesheets  # Unique ID required to update existing cronjob: do not modify.
+  #   user: umcg-labgnkbh-ateambot
+  #   machines: "{{ groups['chaperone'] | default([]) }}"
+  #   minute: '*/5'
+  #   disabled: true
+  #   job: /bin/bash -c "{{ configure_env_in_cronjob }};
+  #        module load NGS_Automated/{{ group_module_versions['umcg-labgnkbh']['NGS_Automated'] }}-bare;
+  #        moveAndCheckSamplesheets.sh -g umcg-labgnkbh"
```
Comment (kdelange) [inline group_vars/copperfist_cluster/vars.yml:761]: in de wingedhelix vars staan er ook nog volgens -d dat47 bij, mis ik die hier? en misschien ook in de betabarrel vars?

File: group_vars/wingedhelix_cluster/vars.yml [hunk 19]
```diff
@@ -708,63 +767,49 @@ crontabs:
...
+         moveAndCheckSamplesheets.sh -g umcg-patho -d dat37"
   - name: NGS_Automated_notifications  # Unique ID required to update existing cronjob: do not modify.
     user: umcg-patho-dm
     machines: "{{ groups['chaperone'] | default([]) }}"
     minute: '*/10'
     disabled: true
     job: /bin/bash -c "{{ configure_env_in_cronjob }};
          module load NGS_Automated/{{ group_module_versions['umcg-patho']['NGS_Automated'] }}-bare;
-         notifications.sh -g umcg-patho -e"
+         notifications.sh -g umcg-patho -e -d dat37 -p prm37"
```
Comment (kdelange) [inline group_vars/wingedhelix_cluster/vars.yml:804]: zitten die -d dat37 -p prm37 wel in de betabarrel en copperfist vars.yml? of hoeft dat niet?

File: single_role_playbooks/pre_deploy_checks.yml [hunk 20]
```diff
@@ -1,4 +1,4 @@
----
+\---
```
Comment (pneerincx) [inline single_role_playbooks/pre_deploy_checks.yml]: What is the `\` doing here?

File: group_vars/betabarrel_cluster/vars.yml [hunk 21]
```diff
@@ -587,75 +596,75 @@ crontabs:
...
+#   - name: NGS_Automated_notifications  # Unique ID required to update existing cronjob: do not modify.
+#     user: umcg-gap-dm
+#     machines: "{{ groups['chaperone'] | default([]) }}"
+#     minute: '*/10'
+#     job: /bin/bash -c "{{ configure_env_in_cronjob }};
+#          module load NGS_Automated/{{ group_module_versions['umcg-gap']['NGS_Automated'] }}-bare;
+#          notifications.sh -g umcg-gap -e"
  ######################################################################################################################
   # umcg-gst group
   ######################################################################################################################
```
Comment (pneerincx) [inline group_vars/betabarrel_cluster/vars.yml]: Please keep correct indentation also for comments.

File: group_vars/wingedhelix_cluster/vars.yml [hunk 22]
```diff
@@ -426,24 +426,30 @@ crontabs:
...
-    job: /bin/bash -c "{{ configure_env_in_cronjob }};
-         module load NGS_Automated/{{ group_module_versions['umcg-gd']['NGS_Automated'] }}-bare;
-         trendanalysis.sh -g umcg-gd"
   - name: NGS_Automated_copyQCDataToTmp  # Unique ID required to update existing cronjob: do not modify.
     user: umcg-gd-dm
     machines: "{{ groups['chaperone'] | default([]) }}"
-    minute: '*/10'
+         weekday: '0'
+         minute: '0'
+         hour: '6'
```
Comment (pneerincx) [inline group_vars/wingedhelix_cluster/vars.yml:434]: wrong indentation

File: group_vars/wingedhelix_cluster/vars.yml [hunk 23]
```diff
@@ -426,24 +426,30 @@ crontabs:
...
     job: /bin/bash -c "{{ configure_env_in_cronjob }};
          module load NGS_Automated/{{ group_module_versions['umcg-gd']['NGS_Automated'] }}-bare;
          copyQCDataToTmp.sh -g umcg-gd"
+  - name: NGS_Automated_trendAnalyse  # Unique ID required to update existing cronjob: do not modify.
+    user: umcg-gd-ateambot
+    machines: "{{ groups['user_interface'] | default([]) }}"
+        weekday: '0'
+        minute: '0'
+        hour: '12'
```
Comment (pneerincx) [inline group_vars/wingedhelix_cluster/vars.yml:443]: wrong indentation

File: group_vars/wingedhelix_cluster/vars.yml [hunk 24]
```diff
@@ -426,24 +426,30 @@ crontabs:
...
+    job: /bin/bash -c "{{ configure_env_in_cronjob }};
+         module load NGS_Automated/{{ group_module_versions['umcg-gd']['NGS_Automated'] }}-bare;
+         trendAnalyse.sh -g umcg-gd"
   - name: NGS_Automated_copyTrendAnalysisDataToPrm  # Unique ID required to update existing cronjob: do not modify.
     user: umcg-gd-dm
     machines: "{{ groups['chaperone'] | default([]) }}"
-    minute: '*/10'
+        weekday: '0'
+        minute: '0'
+        hour: '18'
```
Comment (pneerincx) [inline group_vars/wingedhelix_cluster/vars.yml:452]: wrong indentation

File: group_vars/wingedhelix_cluster/vars.yml [hunk 25]
```diff
@@ -708,63 +793,49 @@ crontabs:
...
+         notifications.sh -g umcg-labgnkbh -e -d dat47 -p prm47"
   - name: NGS_Automated_copyRawDataToPrm_inhouse  # Unique ID required to update existing cronjob: do not modify.
     user: umcg-labgnkbh-dm
     machines: "{{ groups['chaperone'] | default([]) }}"
     minute: '*/10'
     disabled: true
     job: /bin/bash -c "{{ configure_env_in_cronjob }};
          module load NGS_Automated/{{ group_module_versions['umcg-labgnkbh']['NGS_Automated'] }}-bare;
-         copyRawDataToPrm.sh -g umcg-labgnkbh -s porch+wingedhelix"
+         copyRawDataToPrm.sh -g umcg-labgnkbh -s -d dat47 {{ groups['jumphost'] | first}}+{{ groups['user_interface'] | first }}"
```
Comment (pneerincx) [inline group_vars/wingedhelix_cluster/vars.yml:811]:
```suggestion
         copyRawDataToPrm.sh -g umcg-labgnkbh -d dat47 -s {{ groups['jumphost'] | first}}+{{ groups['user_interface'] | first }}"
```

File: group_vars/wingedhelix_cluster/vars.yml [hunk 26]
```diff
@@ -775,28 +846,28 @@ public_sources: [
...
+#  'gonl',
   'inSilico',
-  'snpEff-4.3',
+#  'snpEff-4.3',
 ]
 dynamic_easyconfigs: >-
   {%- set easyconfigs_for_group_modules = [] %}
   {%- for group, modules in group_module_versions.items() %}
-    {%- for software, version in modules.items() %}
+    {%- for software, version in modules.items() %}
```
Comment (pneerincx) [inline group_vars/wingedhelix_cluster/vars.yml:870]:
```suggestion
    {%- for software, version in modules.items() %}
```

File: group_vars/wingedhelix_cluster/vars.yml [hunk 27]
```diff
@@ -813,11 +884,6 @@ dynamic_easyconfigs: >-
     {%- endfor %}
   {%- endfor %}
   {{- easyconfigs_for_group_modules | default([]) -}}
-static_easyconfigs:
-  - c/cluster-utils/cluster-utils-v21.05.1-GCCcore-7.3.0.eb
-  - d/depad-utils/depad-utils-v19.10.1.eb
-  - n/NGS_DNA/NGS_DNA-4.0.5-foss-2018b.eb
```
Comment (pneerincx) [inline group_vars/wingedhelix_cluster/vars.yml:819]: `n/NGS_DNA/NGS_DNA-4.0.5-foss-2018b.eb` is outdated and can be removed, but the latest version of `cluster-utils` and `depad-utils` must be included.

File: group_vars/wingedhelix_cluster/vars.yml [hunk 28]
```diff
@@ -813,11 +884,6 @@ dynamic_easyconfigs: >-
...
 all_easyconfigs: "{{ dynamic_easyconfigs | default([]) | union(static_easyconfigs | default([])) }}"
 easyconfigs: "{{ all_easyconfigs }}"
-chaperone_only_easyconfigs: "{{ all_easyconfigs | select('search', 'NGS_Automated-.*-bare.eb') | list }}"
```
Comment (pneerincx) [inline group_vars/wingedhelix_cluster/vars.yml:822]: We need `NGS_Automated` on chaperone machines
and we should not try to install all other modules there too, because chaperones do not have a dedicated /apps partition and will run out of space on the operating disk if you try to deploy everything; why was this removed?

File: group_vars/wingedhelix_cluster/vars.yml (hunk 19 above)
Comment (pneerincx) [reply group_vars/wingedhelix_cluster/vars.yml:830]: Ja die zitten daar al in.

File: group_vars/wingedhelix_cluster/vars.yml [hunk 29]
```diff
@@ -317,389 +352,439 @@ crontabs:
...
+  - name: NGS_Automated_PullAndProcessGsAnalysisData  # Unique ID required to update existing cronjob: do not modify.
     user: umcg-gst-ateambot
     machines: "{{ groups['user_interface'] | default([]) }}"
     minute: '*/10'
+    disabled: true
     job: /bin/bash -c "{{ configure_env_in_cronjob }};
-         module load NGS_Automated/{{ group_module_versions['umcg-gst']['NGS_Automated'] }}-bare;
-         processGsRawData.sh -g umcg-gst"
-  - name: NGS_Automated_moveSamplesheets  # Unique ID required to update existing cronjob: do not modify.
+         module load NGS_Automated/{{ group_module_versions['umcg-genomescan']['NGS_Automated'] }}-bare;
```
Comment (pneerincx) [inline group_vars/wingedhelix_cluster/vars.yml:715]: genomescan group cronjobs staan hieronder. Dit is gst groep; copy-paste foutje?

File: group_vars/wingedhelix_cluster/vars.yml (hunk 28 above)
Comment (RoanKanninga) [reply group_vars/wingedhelix_cluster/vars.yml:822]: geen idee, foutje denk ik

File: group_vars/betabarrel_cluster/vars.yml [hunk 30]
```diff
@@ -313,458 +345,494 @@ crontabs:
...
+  #        module load NGS_Automated/{{ group_module_versions['umcg-labgnkbh']['NGS_Automated'] }}-bare;
+  #        notifications.sh -g umcg-labgnkbh -e -d dat45 -p prm45"
+  # - name: NGS_Automated_copyRawDataToPrm_inhouse  # Unique ID required to update existing cronjob: do not modify.
+  #   user: umcg-labgnkbh-dm
+  #   machines: "{{ groups['chaperone'] | default([]) }}"
+  #   minute: '*/10'
+  #   disabled: true
+  #   job: /bin/bash -c "{{ configure_env_in_cronjob }};
+  #        module load NGS_Automated/{{ group_module_versions['umcg-labgnkbh']['NGS_Automated'] }}-bare;
+  #        copyRawDataToPrm.sh -g umcg-labgnkbh -s -d dat45 {{ groups['jumphost'] | first}}+{{ groups['user_interface'] | first }}"
```
Comment (pneerincx) [inline group_vars/betabarrel_cluster/vars.yml:808]:
```suggestion
  #        copyRawDataToPrm.sh -g umcg-labgnkbh -d dat45 -s {{ groups['jumphost'] | first}}+{{ groups['user_interface'] | first }}"
```

File: group_vars/copperfist_cluster/vars.yml [hunk 31]
```diff
@@ -313,458 +345,494 @@ crontabs:
...
+  #        module load NGS_Automated/{{ group_module_versions['umcg-labgnkbh']['NGS_Automated'] }}-bare;
+  #        notifications.sh -g umcg-labgnkbh -e -d dat46 -p prm46"
+  # - name: NGS_Automated_copyRawDataToPrm_inhouse  # Unique ID required to update existing cronjob: do not modify.
+  #   user: umcg-labgnkbh-dm
+  #   machines: "{{ groups['chaperone'] | default([]) }}"
+  #   minute: '*/10'
+  #   disabled: true
+  #   job: /bin/bash -c "{{ configure_env_in_cronjob }};
+  #        module load NGS_Automated/{{ group_module_versions['umcg-labgnkbh']['NGS_Automated'] }}-bare;
+  #        copyRawDataToPrm.sh -g umcg-labgnkbh -s -d dat46 {{ groups['jumphost'] | first}}+{{ groups['user_interface'] | first }}"
```
Comment (pneerincx) [inline group_vars/copperfist_cluster/vars.yml]:
```suggestion
  #        copyRawDataToPrm.sh -g umcg-labgnkbh -d dat46 -s {{ groups['jumphost'] | first}}+{{ groups['user_interface'] | first }}"
```

**Review by pneerincx — APPROVED** [review]
